USF-3949: SCP-4.8.18 and SCP-B2B-1.0.19 related mutations and queries updated (#553)
AdobeDocs/commerce-webapi: src/pages/graphql/schema/b2b/requisition-list/mutations/share-requisition-list-by-token.md

---
title: shareRequisitionListByToken mutation
edition: saas
keywords:
  - B2B
---

import CommerceOnly from '/src/_includes/commerce-only.md'

<CommerceOnly />

# shareRequisitionListByToken mutation

<InlineAlert variant="info" slots="text1" />

This mutation is part of the B2B Storefront Compatibility Package and is only available on [Adobe Commerce as a Cloud Service](https://experienceleague.adobe.com/en/docs/commerce/cloud-service/overview).

> The `shareRequisitionListByToken` mutation enables B2B customers to share a Requisition List with colleagues within the same company via a shareable link generated on the storefront. The mutation requires `requisitionListUid` as input parameter. This mutation returns a token associated with the Requisition list which is being shared.

Reviewer: ```suggestion
The `shareRequisitionListByToken` mutation enables B2B customers to share a requisition list with colleagues within the same company using a shareable link generated on the storefront. The mutation requires `requisitionListUid` as an input parameter. This mutation returns the token assigned to the shared requisition list.
```

## Syntax

```graphql
{
    shareRequisitionListByToken(
        requisitionListUid: ID!
    ): ShareRequisitionListByTokenOutput
}
```

## Example usage

> The following example provides a token for a requisition list which is being shared.

Reviewer: ```suggestion
The following example provides a token for a shareable requisition list.
```
